Ratings_MVP/Ratings_MVP/SearachFilmScreen/Presenter/MoviesListPresenterProtocol.swift

```diff
@@ -5,12 +5,14 @@ import Foundation
 
 /// Protocol to MoviesListPresenter
 protocol MoviesListPresenterProtocol: AnyObject {
-    var imageService: ImageServiceProtocol { get }
     var movies: [Movie]? { get set }
     var currentMoviesType: MovieType { get set }
     func fetchFilmInformation(listFilmUrl: String)
     func loadMoviesListFromType(identifier: String?)
     func onTapMovie(movie: Movie)
     func fetchMovies(url: String)
     func getKeyChain() -> KeyChainServiceProtocol
```

Comment:
```suggestion
    func getKeyChain() -> KeyChainServiceProtocol
```
а этот метод еще актуален?